std_msgs/Header header
morefusion_panda_ycb_video/ObjectClass[] classes
